Extension/package.json

```diff
@@ -135,7 +135,8 @@
           "type": "string",
           "enum": [
             "Enabled",
-            "Disabled"
+            "Disabled",
+            "MissingIncludesOnly"
           ],
           "default": "Enabled",
           "description": "Controls whether suspected compile errors detected by the IntelliSense engine will be reported back to the editor. Warnings about #includes that could not be located will always be reported to the editor. This setting is ignored by the Tag Parser engine.",
```

Comment:
We'll need to update the description and change the default to the new enum value that we're adding.